app/controllers/verify/finance_controller.rb

```diff
@@ -9,7 +9,10 @@ def new
     end
 
     def create
-      if step.complete
+      result = step.submit
```

Comment:
There were 2 reasons for this change: 
1) I could not refactor the financials portion by itself by having it call `complete`, which is still present in the parent `Step` class because other steps still need it.
2) For consistency, based on Zach's comments in Slack.  

We can either agree to use different method names in different situations, or try to find something that works universally, like `call`, which is a common choice I've seen for Service Objects.